include/grpc++/security/credentials.h

```diff
@@ -165,6 +168,24 @@ std::shared_ptr<Credentials> CompositeCredentials(
 /// Credentials for an unencrypted, unauthenticated channel
 std::shared_ptr<Credentials> InsecureCredentials();
 
+// User defined metadata credentials.
+class MetadataCredentialsPlugin {
+ public:
+  virtual ~MetadataCredentialsPlugin() {}
+
+  // If this method returns true, the Process function will be scheduled in
+  // a different thread from the one processing the call.
+  virtual bool IsBlocking() const { return true; }
+
+  // Gets the auth metatada produced by this plugin. */
```

Comment:
Should it say "Sets the auth metadata ..." instead? The ```*/``` at the end of line is redundant.